trailing_spaces.py

```diff
@@ -43,7 +43,7 @@ def plugin_loaded():
     trailing_spaces_live_matching = bool(ts_settings.get("trailing_spaces_enabled",
                                          DEFAULT_IS_ENABLED))
     current_highlighting_scope = ts_settings.get("trailing_spaces_highlight_color",
-                                                 "invalid")
+                                                 "region.redish")
```

Comment:
Using `invalid region.redish` would provide a fallback for older versions where colorish scopes are not available.

Reply:
It does look different though: 
`region.redish`:
![Screenshot 2020-03-28 at 17 16 37](https://user-images.githubusercontent.com/153197/77827790-fbbee680-7117-11ea-913c-bfeecc0e9007.png)
`invalid region.redish`:
![Screenshot 2020-03-28 at 17 16 56](https://user-images.githubusercontent.com/153197/77827802-09746c00-7118-11ea-8ce3-0f8e3c5d9dd1.png)

I like the more vibrant look of the former.

For compatibility's sake, I would be able to sacrifice that look but then I've just made this package ST3+ only so it wouldn't really make any difference in practice.

Reply:
:man_shrugging: I was just making you aware of the option. You can do whatever you like. ST2 is dead anyway, but I don't know how many are on ST3<3.2. Probably not a sizeable amount.

Reply:
I appreciate your suggestion but will go with ST3.2+ only behavior since the package was changed to only be compatible with it.